spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe 'User show page', type: :system do
```

Comment:
* I suggest you make Specs file for the integration tests type is `feature` for example ` describe 'the signin process', type: : feature do`